docs/install_linux_mac.rst

```diff
@@ -286,8 +286,10 @@ Or, to install with PostgreSQL support:
 .. note::
 
   To install the development version, replace ``Red-DiscordBot`` in the above commands with the
-  following link:
-
+  link below. Please note that **dev is experimental and not intended for end users.** We will not
```

Comment:
This sentence should be reworded to something along the lines of "The development version of the bot contains experimental changes. It is not intended for normal use." (all in bold)